sidebars.js

```diff
@@ -130,6 +130,26 @@ module.exports = {
           ],
         },
       ],
+      Optimize: [
+        "components/optimize/what-is-optimize",
+        {
+          "User guide": [
+            "components/optimize/userguide/collections-dashboards-reports",
+            "components/optimize/userguide/data-sources",
+            "components/optimize/userguide/creating-dashboards",
+            "components/optimize/userguide/creating-reports",
+          ],
+          "Process analysis": [
+            "components/optimize/userguide/process-analysis/overview",
+            "components/optimize/userguide/process-analysis/outlier-analysis",
+            "components/optimize/userguide/process-analysis/branch-analysis",
+          ],
+          "Additional features": [
+            "components/optimize/userguide/additional-features/filters",
+            "components/optimize/userguide/additional-features/footer",
+          ],
```

Comment:
I think you meant to have the **Process analysis** and **Additional features** within the **User guide**, is that correct? If so, this isn't quite right in the sidebar structure.
<img width="290" alt="Screen Shot 2021-12-17 at 4 08 39 PM" src="https://user-images.githubusercontent.com/1051363/146613646-2c76334b-ec7f-4bfa-8fa0-fae0db39a9e2.png">